app/views/mfa_confirmation/new.html.erb

```diff
@@ -4,8 +4,7 @@
 
 <p>
   <%# for follow up: translate factor_to_change (LG-5701) %>
```

Comment:
I'm going to resurface this with the team for prioritization, since it seems to have been caught up in the Great Icebox-ing purge.

Reply:
I think it'd also be a good opportunity to simplify / remove `factor_from_controller_name`, since (as evidenced by this pull request) it's easy for that mapping to fall out of sync with controllers extending `ReauthnController`.

Reply:
+1000 for removing `factor_from_controller_name`